src/status_im/utils/contenthash.cljs

```diff
@@ -55,12 +55,14 @@
                            hex/decode
                            b58/encode))})))
 
-(defn url [hex]
+(defn url-fn [hex]
```

Comment:
Might be better to call it `url` and memoized one `memoized-url`, it will be explicit

Reply:
we want to use only memoized version

Reply:
I understand, anyway line `(defn url-fn` looks weird to me. :) 
Then let's at least define `url-fn` via `defn-` or just use anonymous function in `url`.